### ISSUE TITLE
Donation button

### ISSUE THREAD

**Issue by BoscoDomingo** [futo-org/android-keyboard#387]
### Use Cases

People who don't have as much to pay, or who want to donate more than the pre-defined amount could benefit greatly.

### Proposal

Have a donation button, so people pay as much or as little as they deem the product is worth, allowing also for multiple payments if so desired to keep sponsoring the project over the long-term

**Comment by edisondotme** [comment]
Yes. Monero specifically. Though I heard that crypto payments including Monero were already on the road map for FUTO.

**Comment by MinuteNewt** [comment]
There really should be a way to donate on a monthly basis. The impact of monthly, recurring donations by loyal and committed users is easy to underestimate. Doing so would be a mistake.

**Comment by genesishep** [comment]
I wanted to donate to the project today and do believe the price is reasonable but sadly I am not currently able to part with the asking price. I very much want to support this project. Please consider adding an option for either flexible amounts or perhaps the ability to split the cost over 2 or 3 monthly installments.

With inflation as it is, those on fixed incomes (ie retirees) may want to support you but are not capable of doing it so easily. Please consider this request. It may sound silly to ask this over such a small amount but I believe there is a huge benefit to "meeting people where they are" so to speak.

I also support the option to allow reoccurring payments for those who are able to offer your project continued support via subscription. Why wouldn't you? As long as a subscription remains optional and signing up is: not default AND clearly stated in a way to avoid confusion it seems to me to be a no brainer. 🙂

I realize I, and others, could simply continue to use FUTO for free however I feel this project is sorely needed... So allow us to support it however we may be able to! 

Thank you!

**Comment by Zvonimir-FUTO** [comment]
We believe we are charging a fair price for a license, if you want to help us out more you can spread the word or buy your friends licenses, otherwise if you can't afford it you're still welcome to continue using it and we're committed to not paywall any functionality.
